src/site/content/en/blog/web-ar/index.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "Augmented reality: You may already know it"
+subhead: If you've used the WebXR Device API already, you're most of the way there.
+authors:
+  - joemedley
+date: 2020-02-12
+hero: hero.jpg
+alt: A person using augmented reality with a smartphone.
+description:
+  If you've already used the WebXR Device API, you'll be happy to know there's
+  very little new to learn. Entering a WebXR session is largely the same.
+  Running a frame loop is largely the same. The differences lie in
+  configurations that allow content to be shown appropriately for augmented
+  reality.
+tags:
+  - post
+  - augmented-reality
+  - virtual-reality
+  - webxr
+---
+
+The WebXR Device API shipped last fall in Chrome 79. As stated then, Chrome's
+implementation of the API is a work in progress. Chrome is happy to announce
+that some of the work is finished. In Chrome 81, two new features have arrived:
+
+* [Augmented reality session types](https://www.chromestatus.com/features/5450241148977152).
+* [Hit testing](https://www.chromestatus.com/features/4755348300759040).
+
+This article covers augmented reality. If you've already used the WebXR Device
+API, you'll be happy to know there's very little new to learn. Entering a WebXR
+session is largely the same. Running a frame loop is largely the same. The
+differences lie in configurations that allow content to be shown appropriately
+for augmented reality. If you're not familiar with the basic concepts of WebXR,
+you should read my earlier posts on the WebXR Device API, or at least be
+familiar with the topics covered therein. You should know how to [request and
+enter a session](https://web.dev/vr-comes-to-the-web/) and you should know how
+to run [a frame loop](https://web.dev/vr-comes-to-the-web-pt-ii).
+
+For information on hit testing, see the companion article [article
+title](article url). The code in this article is based on the Immersive AR
+Session sample
+([demo](https://immersive-web.github.io/webxr-samples/immersive-ar-session.html)
```

Comment:
```suggestion
([demo](https://immersive-web.github.io/webxr-samples/immersive-ar-session.html),
```

FYI @jpmedley if you click the **Commit Suggestion** button you can apply the commit that fixes this from the GitHub UI.